LightBulb/Models/PowerState.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace LightBulb.Models.PowerState
```

Comment:
I think we can greatly simplify the whole thing and remove `PowerState`/`Display` because we don't really care if it's `On`/`Off`/`Dimmed`, we can just refresh gamma on all of these.